README.md

```diff
@@ -29,3 +29,29 @@ resources:
 outputs:
   originURL: ${site.originURL}
 ```
+
+## Input Properties
+
+This component takes the following inputs.
+
+- sitePath (string) - the root directory containing the website's contents to be served (required)
+- withCDN (boolean) - provision a CDN to serve content
+- error404 (string) - the default 404 error page
+- index.html (string) - the default document for the site. Defaults to index.html
+- withCustomDomain (boolean) - Provision a custom domain to serve the site from. This will require a you to set the domainResourceGroup property to the name of the resource group your domain is attached to, as well as the dnsZoneName property for the name of the DNS zone, configured in Azure
+- domainResourceGroup (string) - The name of the resource group your domain is attached to
+- dnsZoneName (string) - The name of the DNS zone
```

Comment:
It's been a bit since I worked on this, but is this the domain name (like `example.come`)? If so, should we just call it `domain` or `domainName`?

Reply:
Yeah, I thought about this as well, since the zone name is essentially whatever the domain name is. I went with `dnsZoneName`, since I felt it would make it more obvious to the user that the dns zone being specified here is an azure resource they would need to configure themselves before running with this option, and once it is configured they just need to pass the name of the dns zone. I should probably improve the description though.